doc/source/tutorial/index.rst

```diff
@@ -28,6 +68,33 @@ SciPy User Guide
    ndimage
    io
 
+.. note::
+
+   In this documentation, we will often assume that the main packages (NumPy,
+   SciPy, and Matplotlib) have been imported as::
+
+       >>> import numpy as np
+       >>> import matplotlib as mpl
+       >>> import matplotlib.pyplot as plt
+
+   While we obviously don't require you to follow these conventions in your own
+   code, they are highly recommended.
```

Comment:
I would maybe remove this paragraph since we want all code samples to be self sufficient now.